Google.PowerShell/Common/ActiveUserToken.cs

```diff
@@ -12,37 +14,131 @@ public class ActiveUserToken
         /// <summary>
         /// The current active user that corresponds to this token.
         /// </summary>
-        private string activeUser = CloudSdkSettings.GetSettingsValue("account");
+        private string activeUser;
 
-        /// <summary>Gets or sets the access token issued by the authorization server.</summary>
-        [Newtonsoft.Json.JsonPropertyAttribute("access_token")]
-        public string AccessToken { get; set; }
+        /// <summary>
+        /// This function will return the current active user.
+        /// </summary>
+        internal Func<string> getActiveUser;
```

Comment:
Is it possible to make this simply `private`? Same with the setter on `AccessToken` and others.